internal/injector/builtin/generator/doc.tmpl

```diff
@@ -19,10 +19,10 @@
         {{ `{{<iconSVG "search-circle">}}` }} Join Point
       </span>
       <div class="root hextra-card-subtitle hx-font-normal hx-px-4 hx-mb-4 hx-mt-2">
-        {{- "\n" }}{{ render .Aspect.JoinPoint | trim }}
+        {{- "\n" }}{{ render .JoinPoint | trim }}
       </div>
     </div>
-    {{- range .Aspect.Advice }}
+    {{- range .Advice }}
```

Comment:
Was this broken ? Where was the templace execution changes in this PR ?

Reply:
The underlying struct is no longer nested because the `ID` field was moved up to the `aspect.Aspect` struct.